macros/base_select.sql

```diff
@@ -42,6 +43,7 @@
     event_server_timestamp_offset,
     user_id,
     user_pseudo_id,
+    to_base64(md5(concat(user_pseudo_id, stream_id))) as client_key,
```

Comment:
@TannerHopkins we do all other key creation in `stg_ga4__events`. Could we move that there to a CTE early in the model?